src/validation_pipeline.ipynb

```diff
@@ -107,7 +108,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "proj_data_path = Path.cwd().parent / \"data\" / \"synthetic_data\"\n",
+    "proj_data_path = Path.cwd().parent / \"data\" / \"synthetic_data_2\"\n",
```

Comment:
What is synthetic_data_2? Perhaps we can give it a better name? Does the old one still work, some explanation here would be helpful.

Reply:
Renamed to "test_data"